lib/podio/models/item.rb

```diff
@@ -187,8 +187,11 @@ def find_field_top(field_id, options={:limit => 8})
     end
 
     # @see https://developers.podio.com/doc/items/get-item-count-34819997
-    def item_count(app_id)
-      Podio.connection.get("/item/app/#{app_id}/count/").body
+    # options are either filters {gender: 1, firstname: "Joe"} or a view {view_id: 123}
+    def item_count(app_id, options = {})
+      Podio.connection.get { |req|
+        req.url("/item/app/#{app_id}/count", options)
+      }.body
```

Comment:
This looks good except please follow the pattern of the other methods for consistency. Assign the response to a `response` variable and implicitly return `response.body`